src/tests/Microsoft.PowerFx.Interpreter.Tests/RecalcEngineTests.cs

```diff
@@ -767,6 +764,21 @@ public void TestNumberBinding()
             Assert.True(recalcEngine.DefineFunctions("Foo(): String => 10;").Errors.Any());
         }
 
+        [Fact]
+        public void TestWithTimeZoneInfo()
+        {
+            var pfxConfig = new PowerFxConfig(new CultureInfo("ja-JP"));
+            var recalcEngine = new RecalcEngine(pfxConfig);
+            var symbols = new SymbolValues();
+
+            symbols.AddService(TimeZoneInfo.FindSystemTimeZoneById("Romance Standard Time")); // (UTC+01:00) Brussels, Copenhagen, Madrid, Paris            

```

Comment:
>.AddService(TimeZoneInfo.FindSystemTimeZoneById("Romance Standard Time"));  [](http://example.com/codeflow?start=19&length=75)

Nit ... since TimeZoneInfo is so critical to our core functions (ditto for CultureInfo), what do you think of an extension method?

symbols.SetTimeZoneInfo(TimeZoneInfo x)

Reply:
Good idea
Could also have multiple extension methods